.gitignore

```diff
@@ -7,4 +7,5 @@ dist
 .coverage
 .tox
 .*cache
-htmlcov
\ No newline at end of file
+htmlcov
+.vs/
```

Comment:
Could you remove this change please? The Prometheus project generally keeps editor specific files out of the .gitignore files, encouraging users to use their global .gitignore instead.

Reply:
done!